### PR TITLE
Auction#show view model

### PR DESCRIPTION
**Pull request by jessieay**
* ALL THE METHODS WE NEED
* IN ONE PLACE

### CODE REVIEW

File: app/presenters/pluralize.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Pluralize
```
Comment (harrisj) [inline app/presenters/pluralize.rb]: Are we reinventing ActiveSupport's `Inflector` functionality here?

File: app/views/auctions/multi_bid/_auction_status.html.erb [hunk 2]
```diff
@@ -1,8 +0,0 @@
-<span class="bid-amount"><%= @view_model.highlighted_bid_amount_as_currency %></span>
```
Comment (harrisj) [inline app/views/auctions/multi_bid/_auction_status.html.erb]: Yay on this

File: app/presenters/pluralize.rb (hunk 1 above)
Comment (jessieay) [inline app/presenters/pluralize.rb]: I don't think so, we're re-using ActionView's `pluralize` which I thin kis exactly what we want (pass in number and word and get pluralized or non-plurialized result depending) http://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize

**Comment by harrisj** [comment]
Merging, but I would like to do something about the complexity of some of these methods in a future PR

**Pull request merged**